### ISSUE TITLE
[NEW] pfe-quote

### ISSUE THREAD

**Issue by starryeyez024** [patternfly/patternfly-elements#772]
Proposal for a new component: pfe-quote that follows the [design system mockups](https://xd.adobe.com/spec/3e62e93c-8338-4f31-5040-3b81f0ed5c71-4853/screen/f5c8e80c-83cb-4ae9-8e92-5991c39e738b/Band-quote-light-theme/)

- Do we need to include `<blockquote>` in the light dom or could JS add it, i.e. do we need to solve for non-JS screenreaders?

```
<blockquote cite="https://www.huxley.net/bnw/four.html">
    <p>Words can be like X-rays, if you use them properly—they’ll go through anything. You read and you’re pierced.</p>
    <footer>—Aldous Huxley, <cite>Brave New World</cite></footer>
</blockquote>

```

- Do blockquotes have value for screen-readers?
- Possible variants:
    - Border on one edge
    - Quote icon - left of content when left-aligned; centered above content when content is centered
    - Increased typography sizing / weight
- Possible Slots:
    - Quote
    - Attribution name
    - Attribution details
    - Possible slot for photo ? [future enhancement]

@todo review / compare with Patternfly quote component:
https://pf4.patternfly.org/documentation/core/components/content

**Comment by castastrophe** [comment]
> Do blockquotes have value for screen-readers?

Yes. Readers can jump between blockquote elements using short keys. How much this feature is used, 🤷‍♀️

**Comment by castastrophe** [comment]
https://schema.org/Quotation

**Comment by castastrophe** [comment]
Search the kit for quote to find more examples.

Band quote: https://xd.adobe.com/view/d20a1c17-375c-4c7f-6827-5fe2f8c2d92f-a43b/screen/f5c8e80c-83cb-4ae9-8e92-5991c39e738b

**Comment by castastrophe** [comment]
Settings:
- alignment: left, center
- border accent

**Comment by bennypowers** [comment]
It appears this issue is no longer relevant. Please open a new issue if it is still relevant.
